features/text-decoration-line-blink.yml

```
@@ -0,0 +1,13 @@
+name: "`text-decoration-line: blink`"
+description: "The deprecated `text-decoration: blink` CSS declaration used to make text alternate between being visible and invisible."
```

Let's write these in the usual way (present-tense, not incorporating the "deprecation"):

```suggestion
description: "The `text-decoration: blink` CSS declaration flashes text between visible and invisible."
```